service/worker/archiver/handler.go

```diff
@@ -104,7 +104,7 @@ func handleRequest(ctx workflow.Context, logger log.Logger, metricsClient metric
 	logger = tagLoggerWithRequest(logger, request)
 	ao := workflow.ActivityOptions{
 		ScheduleToStartTimeout: 1 * time.Minute,
-		StartToCloseTimeout:    2 * time.Minute,
+		StartToCloseTimeout:    1 * time.Minute,
```

Comment:
Can we also update ExpirationInterval on delete to be 5 minutes. Now that we deleted the real delete activity I think local should have more retry time.